Resources/views/Profiler/block.html.twig

```diff
@@ -59,6 +59,8 @@
 {% endblock %}
 
 {% block panel %}
+    {% set profiler_markup_version = profiler_markup_version|default(1) %}
```

Comment:
Please add a `NEXT_MAJOR` comment telling this can be removed when Symfony requirement will be bumped to 2.8+.